commercetools-sdk-base/src/main/java/io/sphere/sdk/client/UnauthorizedException.java

```diff
@@ -1,5 +1,10 @@
 package io.sphere.sdk.client;
 
+/**
+ * Unauthorized access to the commercetools platform wither through invalid client credentials or tokens.
```

Comment:
Oops, it should be "Unauthorized access to the commercetools platform with invalid client credentials or tokens."